lively.morphic/text/morph.js

```diff
@@ -3986,9 +3959,10 @@ export class Text extends Morph {
     topBar.setEditMode(topBar.recoverMode, true);
     this.readOnly = this.prevReadOnly;
     this.collapseSelection();
-    this.editorPlugin?.removeFormattingPopUp && this.editorPlugin.removeFormattingPopUp(true);
-    this.iconButton.owner.remove();
-    delete this.iconButton;
+    if (this.editorModeName === 'richText') {
+      this.editorPlugin.removeFormattingPopUp(true);
+      this.editorPlugin.removeIconButton();
+    }
```

Comment:
Can't this be handled via specialized commands that are either supported by the editor plugin or not? Or is there a scenario where the editorPlugin does support the formatting and icon button yet our editor mode is not set to rich text?

Reply:
No, you are right.